docs/CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ AllowSimplePasswords=true
 
 - To migrate on production builds, update the database password set in Database.URL to be 16 - 50 characters without leading or trailing whitespace. URI parsing rules apply to the chosen password - refer to [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) for special character escape rules.
 
+### Removed
+
+- Removed support for sending telemetry to the deprecated Explorer service
```

Comment:
We should probably be explicit here about the config fields that are being removed and that node ops must remove them, since the node will reject a config that still has them and refuse to start.